Package.swift

```diff
@@ -152,7 +165,7 @@ let package = Package(
         .target(
             name: "PackageDescription",
             exclude: ["CMakeLists.txt"],
-            swiftSettings: [
+            swiftSettings: commonExperimentalFeatures + [
```

Comment:
question: should we be using `swift6CompatibleExperimentalFeatures` instead of `commonExperimentalFeatures` in the event a developer is using Swift 6.0.x to build the project?

Reply:
We want these to apply to any toolchain. The `swift6CompatibleExperimentalFeatures` var only exists for targets that currently fail to build with `MemberImportVisibility` on in 6.0.3.

Reply:
Do we understand why those targets fail to build on 6.0.3?   Can we raise an issue to ensure we get this addressed?

Reply:
They don't fail on 6.1, so my assumption is it's already fixed.